sklearn/utils/testing.py

```diff
@@ -149,6 +149,13 @@ def assert_greater_equal(a, b, msg=None):
     assert a >= b, message
 
 
+def assert_no_nan(a, msg=None):
```

Comment:
I think if we check with assert_all_finite then we should check whether 
`X <= -self.skewedness`
instead of
`X < -self.skewedness`

To avoid having some log(0)=infty. I don't have any preferences though.